api/agent/agent.go

```diff
@@ -673,22 +671,28 @@ func (a *agent) runHot(ctxArg context.Context, call *call, tok ResourceToken) {
 			}
 
 			done := make(chan struct{})
+			start := time.Now()
+			call.slots.enterState(SlotQueueIdle)
 			s := call.slots.queueSlot(&hotSlot{done, proto, errC, container, nil})
 
 			select {
 			case <-s.trigger:
+				call.slots.exitStateWithLatency(SlotQueueIdle, uint64(time.Now().Sub(start).Seconds()*1000))
```

Comment:
super nit: every call to `exitStateWithLatency` uses `uint64(time.Now().Sub(start).Second()*1000`, seems like we should take a `time.Duration` to just be `time.Now().Sub(start)` (or alternatively, just take `start`, or something else idk) and do any conversion inside of `exitStateWithLatency`

Reply:
np, I'll fix this with separate pr.